crates/languages/src/bash.rs

```diff
@@ -0,0 +1,17 @@
+use language::ContextProviderWithTasks;
+use task::{TaskTemplate, TaskTemplates, VariableName};
+
+pub(super) fn bash_task_context() -> ContextProviderWithTasks {
+    ContextProviderWithTasks::new(TaskTemplates(vec![
+        TaskTemplate {
+            label: format!("execute '{}'", VariableName::SelectedText.template_value()),
```

Comment:
Nit: should we surface text in the label? With `rerun.reevaluate_context=true` that'd mean we'd spawn a separate tab for each distinct selection; does that make sense?

Reply:
Good question, I am not sure: for now, the label is the only way to know what exactly is being run (albeit it's quite short) and I've thought it's good to rerun different things.

But indeed maybe it's better to avoid such behavior for selections, will remove.

Reply:
Not sure it's possible, as there could be cases like "navigate through codebase, placing `dbg!`s and rerunning the same failing test" along the way.

With our current task context creation, the preference will always use the newer symbol which would not be that "same failing test" anymore.